docs/source/test_network.md

```diff
@@ -252,6 +252,9 @@ chaincode is ready to be used.
 
 After you have used the `network.sh` to create a channel, you can start a
 chaincode on the channel using the following command:
+
+Before installing the Go,typescript or javascript version of the chaincode make sure that the required language compiler is installed on your local enviroment. For using Java chaincode requires gradle and Java.
```

Comment:
Please proofread the updates again for correct grammar and punctuation. Once a clean PR is available, @joshhus could help with any final edits.